src/components/modals/DeleteModal/index.tsx

```diff
@@ -0,0 +1,72 @@
+"use client";
+
+import CustomButton from "~/components/common/Button/button";
```

Comment:
Move the content of this file to /modals/DeleteModal.tsx

Reply:
Done. Thank you